### ISSUE TITLE
Enable Snapshots?

### ISSUE THREAD

**Issue by kevinSuttle** [npm/npmo-installer#100]
Snapshots default to disabled, but I haven't found a way to enable them within the UI client. The snapshot card displays a `cursor: pointer`, but nothing happens when its clicked.

**Comment by kevinSuttle** [comment]
Would be interesting to enable/disable via the CLI, too.

**Comment by bcoe** [comment]
Agreed, the snapshot feature is new to the installer (the team working on the installer is independent from mine) and I would like to wire up snapshots -- I think it could start quite simple with a backup of couchdb and the package assets.

An option already available for backups, is npm On-Site's replication functionality. You can daisy-chain together multiple npm On-Sites with one acting as the primary, the others as hot spares.

**Comment by kevinSuttle** [comment]
Cool. Good to know. Thanks @bcoe.
